agent/containermetadata/parse_metadata.go

```diff
@@ -82,7 +83,13 @@ func parseDockerContainerMetadata(taskARN string, containerName string, dockerCo
 		seelog.Warnf("Failed to parse container metadata for task %s container %s: container has no configuration", taskARN, containerName)
 	}
 
+	if dockerContainer.ContainerJSONBase == nil {
+		return DockerContainerMetadata{
```

Comment:
Could you please add a `Warnf`log here? Also, a unit test covering this case would be good.